starter/clients/HttpClient.cs

```diff
@@ -0,0 +1,28 @@
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Spike.Inventory
+{
+    public static class DfSpikeOrch_HttpStart
```

Comment:
I thought we'd taken the `DfSpikeOrch` out of this.

Reply:
That we have. I believe this class name was changed to `InventoryOrchestrator_HttpStart`

Reply:
Ahhha... point of time... issue...

Reply:
Renamed Class, Function attribute and Namespace. Drops the word Spike from all places including project name.

New namespace - `InventoryManagement`